roles/openshift_metrics/tasks/cleanup.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- name: remove metrics components
+  command: >
+    {{ openshift.common.client_binary }} -n '{{ project }}'
```

Comment:
That is what the original playbook does:

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_metrics/tasks/main.yaml

I don't know if this is the proper way to handle it, so I left that part out for now.